vanilla/vanilla
# fix subcommunities discussions banner title

Closes https://github.com/vanilla/support/issues/1932

**Steps to test:**
- Make sure you have keystone enable as theme and subcommunities
- Access the recent discussions page within a subcommunity
